docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -45,13 +57,17 @@ Modules appear as apps in the SICK AppStudio workspace, provide the key function
 
 In addition, the SICK AppSpace Coding Starter Kit includes templates that represent application-specific combinations of the modules. The templates demonstrate how modules interact and serve as a starting point for further coding. For example, the remote camera template covers a basic image processing application on a SIM with multiple remote cameras and the steps of setup, image acquisition and result communication, where the data processing step can be freely customized at the code level - as well as all other steps of the application and the respective UIs.
```

Comment:
"In addition" seems to refer to a phrase that is two lines above. I did not see the connection immediately. If this does not change geh intended meaning, I would leave it out:
"The SICK AppSpace coding ..."